qm-windowmanager/etc/systemd/system/active-session.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Session Activate Service
+After=network.target dbus.service
+
+[Service]
+ExecStart=/qm_windowmanager/session-activate/qm_windowmanager_activate_session
+Environment=XDG_RUNTIME_DIR=/run/user/0
+Environment=DBUS_SESSION_BUS_ADDRESS=unix:path=/run/dbus/system_bus_socket
+Restart=always
+SecurityLabelType=qm_container_wayland_t
```

Comment:
As this is now not a container, the label should not be required? Although then not sure if SELinux would allow it. Otherwise, we may have to put it back in a container...

Reply:
Yes, I will give it a try tonight to fix that. Thanks for the hint @aesteve-rh